shared/cpptrace/src/src/symbols/symbols_core.cpp

```
@@ -0,0 +1,154 @@
+         #endif
+         #ifdef CPPTRACE_GET_SYMBOLS_WITH_ADDR2LINE
+          return addr2line::resolve_frames(dlframes);
+         #endif
+         #ifdef CPPTRACE_GET_SYMBOLS_WITH_LIBBACKTRACE
+          return libbacktrace::resolve_frames(frames);
+         #endif
+         #ifdef CPPTRACE_GET_SYMBOLS_WITH_NOTHING
+          return nothing::resolve_frames(frames);
+         #endif
+        #endif
+    }
```

Cppcheck (2.10-2)

[error] Found an exit path from function with non-void return type that has missing return statement

(missingReturn)